be/src/olap/rowset/alpha_rowset.cpp

```diff
@@ -52,11 +52,7 @@ OLAPStatus AlphaRowset::init() {
 }
 
 std::unique_ptr<RowsetReader> AlphaRowset::create_reader() {
-    std::vector<SegmentGroup*> segment_groups;
-    for (auto& segment_group : _segment_groups) {
-        segment_groups.push_back(segment_group.get());
-    }
-    return std::unique_ptr<RowsetReader>(new AlphaRowsetReader(_tablet_schema,
+    return std::unique_ptr<RowsetReader>(new AlphaRowsetReader(
```

Comment:
unique_ptr cannot be assigned

Reply:
unique_ptr can be used as returned value